README.md

```diff
@@ -1,5 +1,16 @@
 # Open Sesame: an Open Source Mentorship Platform
 Open Sesame is a platform intended to make contributing to open source less intimidating by matching first-time contributors to experienced mentors.
 
+## React Integration
+This platform makes use of the [React framework](https://reactjs.org/). In order to provide React functionality to the project search page while allowing for non-React HTML and JS
+in other areas of the web app, [React is loaded and run on a page-by-page basis](https://reactjs.org/docs/add-react-to-a-website.html) (see `projects.html` for an example of this).
+* **Be sure to replace all script tags with the React development scripts to the minified production versions** when the platform is being deployed to production.
+These script tags are all clearly commented in each HTML page that makes use of React.
+### JSX Preprocessing
+This platform makes use of React's [JSX](https://reactjs.org/docs/introducing-jsx.html). In order to use JSX, a [preprocessor](https://reactjs.org/docs/add-react-to-a-website.html#add-jsx-to-a-project) must be run alongside development and deployment. This is set up as follows:
+* **JSX preprocessing relies on [NodeJS dependencies](https://reactjs.org/docs/add-react-to-a-website.html#add-jsx-to-a-project)** which can be acquired by running `npm install` from within the outermost directory of the repository.
+* For every directory containing React components, there exists a subdirectory called `jsx-processed`, which contains the JS files that have had their JSX parsed by the preprocessor. These are the JS files that should be imported with script tags.
+* **The parsing of JSX does not happen automatically**, and must instead be started with `npm run jsx-preprocessor` from within the outermost directory of the repository. This must be done before a development server is run or the application is deployed.
```

Comment:
Same here, since the information is repeated. If all these commands need to be run in a certain directory, let's specify it and just put a notice above.

Reply:
Also agreed.